src/avt/VisWindow/Interactors/Navigate3D.C

```diff
@@ -91,8 +91,14 @@ Navigate3D::OnTimer(void)
         break;
 
       case VTKIS_PAN:
+        // Currently the SetWindowCenter called from avtViewInfo.C
+        // does not get used in the vtkOSPRayCamerNode so instead pan
+        // the camera rather than the image.
+#ifdef HAVE_OSPRAY
+        PanCamera3D(Pos[0], Pos[1]);
+#else
         PanImage3D(Pos[0], Pos[1]);
-
+#endif
```

Comment:
If HAVE_OSPRAY is true, the pan behavior is being changed whether we are using OSPRAY or not.  This doesn't seem right.